### PR TITLE
Add wiki build to `make docs` and verify in CI

### LINKED ISSUE #2273: Check wiki for valid links in CI
`mkdocs` makes it easy to catch invalid links to markdown files within the wiki. Running `mkdocs build --clean -s` will exit with a nonzero status if any of the linked files are invalid. This completes in under 1 second on my machine.

Running this on the Jenkins build could help us catch broken links before they make it into the wiki (such as https://github.com/facebook/osquery/pull/2272).

### PR DESCRIPTION
**Pull request by zwass**
Closes #2273

### CODE REVIEW

**Comment by osqueryer** [comment]
Build origin/pr/2279/merge (Job: 2917) failed :-1:

**Comment by zwass** [comment]
We need to run `make deps` or install `mkdocs` on the test boxes:
```
/bin/sh: 1: mkdocs: not found
make[4]: *** [CMakeFiles/docs] Error 127
make[3]: *** [CMakeFiles/docs.dir/all] Error 2
make[2]: *** [CMakeFiles/docs.dir/rule] Error 2
make[1]: *** [docs] Error 2
make: *** [docs] Error 2
Build step 'Execute shell' marked build as failure
```

File: tools/lib.sh [hunk 1]
```diff
@@ -138,6 +138,9 @@ function build() {
   # Clean previous build artifacts.
   $MAKE clean
 
+  # Build doxygen and wiki docs
+  $MAKE docs
```
Comment (theopolis) [inline tools/lib.sh]: If we remove this from the default build script I can create a job dedicated to `make docs` for cleaner/quicker output.

Comment (zwass) [inline tools/lib.sh]: Will do. Should I put it somewhere else or will it go into a config file not in the repo?

Comment (theopolis) [inline tools/lib.sh]: Yeah, it will go into the Jenkins configuration.

**Comment by theopolis** [comment]
Awesome, thanks @zwass!

**Pull request merged**